specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/examples/UpdatePrivateStore.json

```diff
@@ -1,11 +1,13 @@
 {
   "parameters": {
-    "PrivateStoreId": "a0e28e55-90c4-41d8-8e34-bb7ef7775406",
+    "privateStoreId": "a0e28e55-90c4-41d8-8e34-bb7ef7775406",
     "api-version": "2020-01-01",
-    "parameters": {
-      "properties": {
-        "availability": "disabled",
-        "eTag": "\"9301f4fd-0000-0100-0000-5e248b350345\""
+    "PrivateStore": {
```

Comment:
> "PrivateStore": { [](start = 3, length = 18)

this breaking change will break existing SDK, is there any justification for this breaking change without a API version change?

Reply:
@qinwens, it wont break it, because we stiil dont have the sdk.
After this merge, we will generate our first sdk